metron-platform/metron-elasticsearch/src/test/java/org/apache/metron/elasticsearch/dao/ElasticsearchMetaAlertDaoTest.java

```diff
@@ -489,4 +505,54 @@ public void testHandleMetaUpdateAlert() throws IOException {
     verify(mockEsDao, times(1)).getLatest(guidAlert, null);
     verify(mockEsDao, times(1)).batchUpdate(updates);
   }
+
+  @Test
+  public void getAllAlertsForMetaAlertShouldReturnAllAlerts() throws Exception {
```

Comment:
Are we able to replicate this defect in the integration tests? Personally, I'd prefer to see the integration tests validate something like this, if at all possible.  That would also help our migration to ES 5.x over this mock driven test.